### PR TITLE
Refactor pi1_val and add user guide

### PR DESCRIPTION
**Pull request by samaloney**

### CODE REVIEW

File: stixcore/tmtc/packets.py [hunk 1]
```diff
@@ -455,6 +473,7 @@ def __init__(self, data, idb=None):
         self.data = parse_variable(self.source_packet_header.bitstream, tree)
         self.group_repeaters()
 
+    @abc.abstractmethod
```
Comment (nicHoch) [inline stixcore/tmtc/packets.py:476]: cool i was thinking as well if there is some kind of abstract

**Review by nicHoch — APPROVED** [review]
looks all good

File: stixcore/idb/tests/test_manager.py [hunk 2]
```diff
@@ -60,8 +60,8 @@ def test_find_version(idb_manager):
     assert len(str(e.value)) > 1
 
     with pytest.raises(ValueError) as e:
-        idb = idb_manager.get_idb(utc=datetime(2018, 6, 6))
-        assert idb.get_idb_version() == "2.26.34"
+        idb = idb_manager.get_idb(utc=datetime(2016, 6, 6))
+        assert idb.version == "2.26.34"
```
Comment (samaloney) [inline stixcore/idb/tests/test_manager.py:64]: @nicHoch I'm not sure why I had to change this - I was getting a IDB not connected error do you seen any issue with this change?

File: docs/users.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    packet_data = []
+    tree = Et.parse('path/to/moc/aaaaPktTmRawbbbbb.xml')
+    root = tree.getroot()
+    for i, node in enumerate(root.iter('Packet')):
+        packet_binary = unhexlify(node.text)
+        # Not sure why guess and extra moc header
+        packet_data.append(packet_binary[76:])
+
+Next the source header, data header and ssid or PI1_VAL can be parsed and filtered. In the following
+code we filter for for server 21, subervice 6 and SSID or 21
```
Comment (nicHoch) [inline docs/users.rst:55]: for service 21, subservice 6

File: stixcore/idb/tests/test_manager.py (hunk 2 above)
Comment (nicHoch) [reply stixcore/idb/tests/test_manager.py:64]: hm i am also not sure why

**Review by nicHoch — APPROVED** [review]
ok will also not use the @property anymore

**Pull request merged**